src/util-var-name.c

```diff
@@ -275,16 +273,17 @@ int VarNameStoreActivate(void)
         SC_ATOMIC_SET(active, new_active);
         SCLogDebug("new store active");
 
-        struct timeval now;
-        memset(&now, 0, sizeof(now));
-        gettimeofday(&now, NULL);
+        struct timeval tv;
+        memset(&tv, 0, sizeof(tv));
```

Comment:
Is the `memset` needed? If yes, it seems to be missing in other places.

Reply:
We have it in other places, see
```
git grep -B1 gettimeofday|grep memset
src/source-ipfw.c-        memset (&IPFWts, 0, sizeof(struct timeval));
src/source-nflog.c-        memset(&tv, 0, sizeof(tv));
src/source-nfq.c-        memset(&tv, 0, sizeof(tv));
src/suricata.c-    memset(&suri->start_time, 0, sizeof(suri->start_time));
src/suricata.c-    memset(&end_time, 0, sizeof(end_time));
src/util-random.c-    memset(&tv, 0, sizeof(tv));
src/util-var-name.c-            memset(&ts, 0, sizeof(ts));
src/util-var-name.c-        memset(&tv, 0, sizeof(tv));
```
However I agree it's probably not needed as there are many more places were it's not done. Not sure how I developed this as a half baked pattern.